plugins/woocommerce/src/Admin/PluginsHelper.php

```diff
@@ -532,4 +534,47 @@ public static function get_activation_status( $job_id = null ) {
 		return self::get_action_data( $actions );
 	}
 
+	/**
+	 * Show notices to connect to woo.com for unconnected store in the plugin list.
+	 *
+	 * @return void
+	 */
+	public static function maybe_show_connect_notice_in_plugin_list() {
+		global $pagenow;
+
+		if ( 'plugins.php' !== $pagenow ) {
+			return;
+		}
+
+		if ( WC_Helper::is_site_connected() ) {
+			return;
+		}
+
+		$notice_string = '';
+
+		if ( count( WC_Helper::get_local_woo_plugins() ) > 0 ) {
+			$notice_string .= __( 'Your store might be at risk as you are running old versions of Woo plugins.', 'woocommerce' );
+			$notice_string .= ' ';
+		}
+
+		$connect_page_url = add_query_arg(
+			array(
+				'page' => 'wc-admin',
+				'tab'  => 'my-subscriptions',
+				'path' => rawurlencode( '/extensions' ),
+			),
+			admin_url( 'admin.php' )
+		);
+
+		$notice_string .= sprintf(
+			/* translators: %s: Connect page URL */
+			__( '<a href="%s">Connect your store</a> to Woo.com to get updates and streamlined support for your subscriptions.', 'woocommerce' ),
+			$connect_page_url
```

Comment:
```suggestion
			esc_url( $connect_page_url )
```

might be not needed if causes double-encode